native/src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -78,12 +95,14 @@ export default (
         return { ...state, languages: { status: 'ready', models: action.params.languages } }
       case 'FETCH_LANGUAGES_FAILED':
         return { ...state, languages: { status: 'error', ...action.params } }
+      case 'REFRESH_CATEGORY':
       case 'PUSH_CATEGORY':
-        return pushCategory(state, action)
+        return pushOrRefreshCategory(state, action)
       case 'PUSH_POI':
         return pushPoi(state, action)
+      case 'REFRESH_EVENT':
       case 'PUSH_EVENT':
-        return pushEvent(state, action)
+        return pushOrRefreshEvent(state, action)
```

Comment:
If you have a boolean in the PUSH_* actions then the name would stay: `pushCategory`